### ISSUE TITLE
Time Trigger Issues

### ISSUE THREAD

**Issue by Y07yoyo** [Azure/azure-functions-host#6433]
### Description

We detected that you have unscheduled function execution(s) which is past due because of host start up.

| TIMESTAMP | FunctionName | Orignial Schedule  |
| :---: | :---: | :---: |
| 7/30/2020 8:10:15 AM | `CampsieReset` | 2020-07-30T06:45:00.0000000+00:00 |
| 7/30/2020 8:10:15 AM | `CanterburyReset` | 2020-07-30T06:45:00.0000000+00:00 |

### Recommended Action

Function executions will be pastdue when the current function invocation is later than scheduled. For example, a function app restart might cause an invocation to be missed.<br /><br />Check **Web App Restart** detector below for more information.

### Explanation of Issue:
Hi, I have a function app that is using a time trigger to run and it is set to run at 6.45am everyday Australian Eastern Time. It was running find for a few days and then suddenly yesterday the function ran at 6pm.

#### Investigative information

Please provide the following:

- Timestamp: 2020-07-31T00:51:34Z
- Function App version: 3.0.14120.0
- Function App name: DDILabs
- Function name(s) (as appropriate): CampsieReset, CanterburyReset
- Invocation ID: ac20af9f-b972-48b1-a9b9-84152031e8e3
- Region: Australia East

#### Repro steps

![Invocation Traces](https://user-images.githubusercontent.com/40599534/88988341-f834a980-d31b-11ea-9240-d75c39110450.PNG)

#### Expected behavior

The function should run at 6.45am AEST everyday. This should be around 8.45pm UTC time. 

#### Actual behavior

Yesterday the function ran at 6.10 pm AEST which is 8.10am UTC

#### Related information 

Provide any related information 

* Programming language used: Python

My Cron Expression is: 0 45 6 * * * 
I have added WEBSITE_TIME_ZONE: Aus Eastern Standard Time in the Application Settings
My app is using Consumption price tier

**Comment by Y07yoyo** [comment]
![information](https://user-images.githubusercontent.com/40599534/88988724-0505cd00-d31d-11ea-9ab8-771385027eab.PNG)

**Comment by v-anvari** [comment]
Hi @Y07yoyo , There has been few fixes made on triggers lately, which should have fixed this issue by now. Let us know if you are still facing the same issue with timer trigger.

We have tried to reproduce the issue, but the timer trigger is working as expected and on time. We also tried to produce the logs for your function app "DDILabs", but seems like the app doesn't exist anymore or has been scaled down.
